sdk/schemaregistry/azure-schemaregistry/swagger/README.md

```diff
@@ -6,11 +6,11 @@
 ```ps
 cd C:\Work\SchemaRegistry\
 autorest --reset
-autorest --use=D:\Projects\autorest.python --low-level-client --modelerfour.lenient-model-deduplication
+autorest --low-level-client --modelerfour.lenient-model-deduplication README.md
 ```
 ### Settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/1e23d91e875e4464e57667639e06408cef99868d/specification/schemaregistry/data-plane/Microsoft.EventHub/preview/2020-09-01-preview/schemaregistry.json
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2021-10/schemaregistry.json
```

Comment:
should this url be "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2021-10/schemaregistry.json" -- the raw file

Reply:
hmm, strangely I generated with the non-raw file and it worked? I can try regenerating with the raw